src/braket/emulation/device_emulator_utils.py

```diff
@@ -0,0 +1,120 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+import json
+
+from braket.device_schema.ionq.ionq_device_capabilities_v1 import IonqDeviceCapabilities
+from braket.device_schema.result_type import ResultType
+from braket.device_schema.standardized_gate_model_qpu_device_properties_v1 import (
+    CoherenceTime,
+    Fidelity1Q,
+    FidelityType,
+    GateFidelity2Q,
+    OneQubitProperties,
+    TwoQubitProperties,
+)
+
+from braket.circuits.translations import BRAKET_GATES
+
+DEFAULT_SUPPORTED_RESULT_TYPES = [
+    ResultType(name="Sample", observables=["x", "y", "z", "h", "i"], minShots=1, maxShots=20000),
+    ResultType(
+        name="Expectation", observables=["x", "y", "z", "h", "i"], minShots=1, maxShots=20000
+    ),
+    ResultType(name="Variance", observables=["x", "y", "z", "h", "i"], minShots=1, maxShots=20000),
+    ResultType(name="Probability", observables=None, minShots=1, maxShots=20000),
+]
+
+
+def standardize_ionq_device_properties_json(device_properties_json: str) -> str:
```

Comment:
Does this need to be public? In fact, is it necessary to have this at all (since it's deep down within capabilities parsing)?